FND/Places/Addresses.rdf

```diff
@@ -159,14 +166,28 @@
 		<fibo-fnd-utl-av:explanatoryNote>Other unconventional addresses may include rural and highway route addresses, general delivery addresses, post office box addresses, private mail center addresses, and so forth.</fibo-fnd-utl-av:explanatoryNote>
 	</owl:Class>
 	
+	<owl:NamedIndividual rdf:about="&fibo-fnd-plc-adr;Department">
+		<rdf:type rdf:resource="&fibo-fnd-plc-adr;SecondaryUnitDesignator"/>
+		<rdfs:label xml:lang="en">department</rdfs:label>
+		<skos:definition>division of a large organization such as a government, university, business, or shop, dealing with a specific subject, commodity, or area of activity</skos:definition>
+		<fibo-fnd-plc-adr:requiresSecondaryUnitRange rdf:datatype="&xsd;boolean">true</fibo-fnd-plc-adr:requiresSecondaryUnitRange>
+	</owl:NamedIndividual>
+	
 	<owl:NamedIndividual rdf:about="&fibo-fnd-plc-adr;Floor">
 		<rdf:type rdf:resource="&fibo-fnd-plc-adr;SecondaryUnitDesignator"/>
-		<rdfs:label>floor</rdfs:label>
+		<rdfs:label xml:lang="en">floor</rdfs:label>
 		<skos:definition>all of the rooms or areas on the same level of a building; a story</skos:definition>
```

Comment:
I think it depends on the building and the way that the building itself manages address details.  For example, the parking garage in the building I park in to visit Wells Fargo has its office in the basement, rather than on a specific floor, in the building, but I would have to check with them to get their formal postal address to know how they are referenced by the post office.